tests/api/test_order.py

```diff
@@ -3162,3 +3149,39 @@ def test_create_invoice_empty_params(
     assert not Invoice.objects.filter(
         order_id=order.pk, status=InvoiceStatus.READY
     ).exists()
+
+
+@mock.patch("saleor.order.emails.send_invoice.delay")
+def test_send_invoice(email_mock, staff_api_client, permission_manage_orders, orders):
```

Comment:
Why are you using `orders` fixture instead of `order`?

Reply:
Because i didn't see it. Will change that here and everywhere else.